packages/planes-ds/src/lib/components/atoms/button/index.tsx

```diff
@@ -9,7 +9,7 @@ export const AT_BUTTON_VARIANT = {
 export type AtButtonVariant = keyof typeof AT_BUTTON_VARIANT;
 
 export const variantClasses: Record<AtButtonVariant, string> = {
-  PRIMARY: "bg-pink-300 hover:bg-green-400 active:bg-pink-500",
+  PRIMARY: "bg-pink-300 hover:bg-red-400 active:bg-pink-500",
```

Comment:
Hey, probaby we can move it to a constant blabla

```suggestion
const color = "bg-pink-300 hover:bg-red-400 active:bg-pink-500"
  PRIMARY:color ,
```